remotespark/wrapperkernel/sparkkernelbase.py

```diff
@@ -135,6 +107,27 @@ def do_shutdown(self, restart):
 
         return self._do_shutdown_ipykernel(restart)
 
+    @staticmethod
+    def _get_code_transformer(subcommand):
```

Comment:
From a structural standpoint I don't know how much sense it makes for this function to be in this file and not in `codetransformers.py`.